### PR TITLE
KAFKA-4956: System test for request quotas

### PR DESCRIPTION
**Pull request by rajinisivaram**

### CODE REVIEW

**Comment by rajinisivaram** [comment]
@junrao Can you review this, please? Thank you...

**Comment by rajinisivaram** [comment]
Started system test run here: https://jenkins.confluent.io/job/system-test-kafka-branch-builder/902/
All tests have passed.

File: tests/kafkatest/tests/client/request_quota_test.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+        if client_id is not None:
+            cmd += " --entity-type clients --entity-name %s" % client_id
+        node.account.ssh(cmd)
+
+
+class RequestQuotaTest(Test):
+    """
+    Tests to verify that clients with small request quota are throttled,
+    clients with large request quota are not throttled and broker's
+    exempt metrics as well as client throttle netrics are updated.
```
Comment (junrao) [inline tests/kafkatest/tests/client/request_quota_test.py:52]: netrics => metrics

**Review by junrao — COMMENTED** [review]
@rajinisivaram : Thanks for the patch. LGTM. I was wondering if this system test adds any additional value than the RequestQuotaTest unit test. It seems that the new system test just further verifies the metrics in the client? If so, perhaps we could just do that in a unit test?

**Comment by rajinisivaram** [comment]
@junrao Thank you for the review. Agree that this system test doesn't really add much value over the unit tests. I have opened PR #3190 to verify client-side throttle time metrics in the existing quota integration test. If you are happy with that, I can close this PR.

**Comment by rajinisivaram** [comment]
As discussed above, client metrics check has been added to integration test, closing this PR since the functionality provided by this system test is already included in the unit/integration tests.

**Pull request closed without merging**